source/core/Events.cpp

```diff
@@ -155,6 +169,10 @@ class EventOracle {
     void DoneProcessingEvent(EventQueueID qID) {
         return _qObject[qID].DoneProcessingEvent();
     }
+    void ClearEventQueue(EventQueueID qID) {
+        if (qID < _qObject.size())
+            _qObject[qID].ClearQueue();
```

Comment:
Can qID be 0 and should we ClearQueue in that case? Is that the kNotAQueueID case?

Reply:
qID is not the kNotAQueueID case.  Calling clear on that qID shouldn't happen because the qID would have had to be valid before to set up the cleanup proc on it, but if it did it would be innocuous because it is unused and already empty.